services/director/tests/unit/test_registry_proxy.py

```diff
@@ -237,17 +253,42 @@ async def test_registry_caching(
     print("time to retrieve services with cache: ", time_to_retrieve_with_cache)
 
 
-@pytest.mark.skip(reason="test needs credentials to real registry")
-async def test_get_services_performance(
-    configure_registry_access: EnvVarsDict,
+@pytest.fixture
+def configure_number_concurrency_calls(
+    app_environment: EnvVarsDict,
+    monkeypatch: pytest.MonkeyPatch,
+) -> EnvVarsDict:
+    return app_environment | setenvs_from_dict(
+        monkeypatch,
+        envs={
+            "DIRECTOR_REGISTRY_CLIENT_MAX_CONCURRENT_CALLS": "50",
+            "DIRECTOR_REGISTRY_CLIENT_MAX_NUMBER_OF_RETRIEVED_OBJECTS": "50",
+        },
+    )
+
+
+def test_list_services_performance(
+    skip_if_external_envfile_dict: None,
+    configure_external_registry_access: EnvVarsDict,
+    configure_number_concurrency_calls: EnvVarsDict,
+    registry_settings: RegistrySettings,
     app: FastAPI,
+    benchmark: BenchmarkFixture,
 ):
-    start_time = time.perf_counter()
-    services = await registry_proxy.list_services(app, registry_proxy.ServiceType.ALL)
-    stop_time = time.perf_counter()
-    print(
-        f"\nTime to run getting services: {stop_time - start_time}s, #services {len(services)}, time per call {(stop_time - start_time) / len(services)}s/service"
-    )
+    async def _list_services():
+        start_time = time.perf_counter()
+        services = await registry_proxy.list_services(
+            app, registry_proxy.ServiceType.ALL
+        )
+        stop_time = time.perf_counter()
+        print(
+            f"\nTime to list services: {stop_time - start_time:.3}s, {len(services)} services in {registry_settings.resolved_registry_url}, rate: {(stop_time - start_time) / len(services or [1]):.3}s/service"
+        )
+
+    def run_async_test() -> None:
+        asyncio.get_event_loop().run_until_complete(_list_services())
+
+    benchmark.pedantic(run_async_test, rounds=5)

```

Comment:
here we have no assertions. that means this tests i just running the benchmark but cannot fail?